File: staffapp/src/main/java/com/example/staffapp/ui/work/SectionHints.kt
package com.example.staffapp.ui.work

object SectionHints {
    fun forSection(key: String): String = when (key) {
        "dashboard" -> "Сводка, задачи, лиды, записи"
        "clients" -> "Список клиентов и карточки"
        "schedule" -> "Тренировки и календарь"
        "bookings" -> "Записи на занятия"
        "leads" -> "Воронка и новые лиды"
        "app_support" -> "Обращения из приложения"
        "finance" -> "Доходы и расходы"
        "crm_staff" -> "Сотрудники CRM"
        "tasks" -> "Задачи и поручения"
        "subscriptions" -> "Абонементы клиентов"
        else -> "Данные раздела CRM"
    }
}

File: MatskocaAdmin/MatskocaAdmin/Labels/SectionHints.swift
import Foundation

enum SectionHints {
    static func forSection(_ key: String) -> String {
        switch key {
        case "dashboard": return "Сводка, задачи, лиды, записи"
        case "clients": return "Список клиентов и карточки"
        case "schedule": return "Тренировки и календарь"
        case "bookings": return "Записи на занятия"
        case "leads": return "Воронка и новые лиды"
        case "app_support": return "Обращения из приложения"
        case "finance": return "Доходы и расходы"
        case "crm_staff": return "Сотрудники CRM"
        case "tasks": return "Задачи и поручения"
        case "subscriptions": return "Абонементы клиентов"
        default: return "Данные раздела CRM"
        }
    }
}
